### ISSUE TITLE
How to solve this problem about compile?

### ISSUE THREAD

**Issue by dangjisheng** [scenenn/pointwise#12]
i am facing this error when i compile "tf_conv3p_compile_.sh". kindly guide me if i am doing something wrong?

$chmod a+x *
$ ./tf_conv3p_compile_.sh
/home/sarshad/.local/lib/python3.5/site-packages/h5py/init.py:36: FutureWarning: Conversion of the second argument of issubdtype from float to np.floating is deprecated. In future, it will be treated as np.float64 == np.dtype(float).type.
from ._conv import register_converters as _register_converters
nvcc warning : The 'compute_20', 'sm_20', and 'sm_21' architectures are deprecated, and may be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
gcc: error: .cu: No such file or directory
gcc: warning: ‘-x c++’ after last input file has no effect
gcc: fatal error: no input files
compilation terminated.
g++: error: .cpp: No such file or directory
g++: fatal error: no input files
compilation terminated.
g++: error: .cpp: No such file or directory
g++: fatal error: no input files
compilation terminated.
In file included from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/mutex.h:31:0,
from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/framework/op.h:32,
from tf_conv3p_grid.cpp:1:
/usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/default/mutex.h:25:22: fatal error: nsync_cv.h: No such file or directory
compilation terminated.
nvcc warning : The 'compute_20', 'sm_20', and 'sm_21' architectures are deprecated, and may be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
In file included from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/mutex.h:31:0,
from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/framework/op.h:32,
from tf_conv3p_naive_ctxmem.cu:1:
/usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/default/mutex.h:25:22: fatal error: nsync_cv.h: No such file or directory
compilation terminated.
In file included from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/mutex.h:31:0,
from /usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/framework/op.h:32,
from register_op.cpp:1:
/usr/local/lib/python3.5/dist-packages/tensorflow/include/tensorflow/core/platform/default/mutex.h:25:22: fatal error: nsync_cv.h: No such file or directory
compilation terminated.
g++: error: tf_conv3p.so_cuda.o: No such file or directory
g++: error: tf_conv3p.so_cpu.o: No such file or directory
rm: cannot remove 'tf_conv3p.so_cpu.o': No such file or directory
rm: cannot remove 'tf_conv3p.so_cuda.o': No such file or directory

**Comment by songuke** [comment]
What is your Tensorflow version? You might want to try this instead: 
https://github.com/scenenn/pointwise/blob/master/tf_ops/conv3p/tf_conv3p_compile_tf14.sh

**Comment by dangjisheng** [comment]
> What is your Tensorflow version? You might want to try this instead:
> https://github.com/scenenn/pointwise/blob/master/tf_ops/conv3p/tf_conv3p_compile_tf14.sh

tensorflow                         1.13.1                   
tensorflow-gpu                     1.9.0                    
When I try conv3p/tf_conv3p_compile_tf14.sh to compile,I meet the same problem,what wrong with me ?

**Comment by STARBOYsachin** [comment]
Have you solved the issue?

**Comment by longmalongma** [comment]
No,I can't solve this problem about compile.so I can't run your code.




------------------ 原始邮件 ------------------
发件人: "sachinAI"<notifications@github.com>;
发送时间: 2019年7月24日(星期三) 下午2:16
收件人: "scenenn/pointwise"<pointwise@noreply.github.com>;
抄送: "longma"<1442342449@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [scenenn/pointwise] How to solve this problem about compile?(#12)




Have you solved the issue?

**Comment by longmalongma** [comment]
No,I can't solve this issue




------------------ 原始邮件 ------------------
发件人: "sachinAI"<notifications@github.com>;
发送时间: 2019年7月24日(星期三) 下午2:16
收件人: "scenenn/pointwise"<pointwise@noreply.github.com>;
抄送: "longma"<1442342449@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [scenenn/pointwise] How to solve this problem about compile?(#12)




Have you solved the issue?

**Comment by GuoliangShi** [comment]
I also meet this error, i think different version result this problem. so i create a new environment with tensorflow 1.9, CUDA 9.0 and it works.
